<!--  for trajectory tracking -->
<launch>

    <arg name="mav_name" default="firefly" />

    <group ns="$(arg mav_name)">

        <node name="lee_position_controller_node" pkg="rotors_control" type="lee_position_controller_node" output="screen">
            <rosparam command="load" file="$(find rotors_gazebo)/resource/lee_controller_$(arg mav_name).yaml" />
            <rosparam command="load" file="$(find rotors_gazebo)/resource/$(arg mav_name).yaml" />
            <remap from="odometry" to="odometry_sensor1/odometry" />
        </node>
    <!--
        <node name="waypoint_publisher" pkg="rotors_gazebo" type="waypoint_publisher" output="screen" args="15 10 15 0 2"/>
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    -->
    <!--
        <node name="waypoint_publisher_move_to_husky" pkg="rotors_gazebo" type="waypoint_publisher_move_to_husky" output="screen" args="15 10 15 0 2"/>
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    -->
    <!--
        <node name="waypoint_publisher_path_planning" pkg="rotors_gazebo" type="waypoint_publisher_path_planning" output="screen" args="15 19 10 0 2"/>
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    -->
    <!--
        <node name="waypoint_publisher_Landing" pkg="rotors_gazebo" type="waypoint_publisher_Landing" output="screen" args="15 10 15 0 2"/>
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
    -->
    
        <node name="path_planning_and_landing" pkg="rotors_gazebo" type="path_planning_and_landing" output="screen" args="15 10 10 0 2"/>
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
      
    </group>

    <!-- <node name="geo" pkg="offb" type="geo" output="screen"/> -->

</launch>
